src/components/Map/markerStyle.tsx

```diff
@@ -17,24 +18,33 @@ export function createSensorIcon(aqiReading: string): H.map.Icon {
   if (aqi < satisfactory) {
     color = '"#1B8DFF"'; // Light blue
   } else if (aqi < moderateConcern) {
-    color = '"#304ACC"'; // Dark blue
+    color = '"#4765f5"'; // Dark blue
   } else if (aqi < sensitiveGroups) {
-    color = '"#852199"'; // Purple
+    color = '"#9247a1"'; // Purple
   } else if (aqi < healthRiskForAll) {
-    color = '"#CC244B"'; // Pink-red
+    color = '"#cc2475"'; // Pink-red
   } else {
     color = '"#FF3628"'; // Red
   }
 
+  // Set marker size
+  const standardMarkerSize = 20;
+  const hoverMarkerSize = 22;
+  const ms = size === 'standard' ? standardMarkerSize : hoverMarkerSize;
+
   // SVG Marker Image
   /* eslint-disable spellcheck/spell-checker */
   const svgMarkup =
-    '<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40">' +
-    '<circle cx="20" cy="20" r="20" fill=' +
+    '<svg xmlns="http://www.w3.org/2000/svg" width="44" height="44">' +
+    '<circle stroke="black" stroke-width="0.5" cx="22" cy="22" r="' +
```

Comment:
Can we add something to the SVG to make it clearer when a sensor is selected? Perhaps a bolder outline or something along those lines